.github/workflows/ci.yml

```diff
@@ -41,21 +44,84 @@ jobs:
           pre-commit run --all-files
           pytest taf/tests
 
+  build_and_upload_wheel:
+    runs-on: ubuntu-latest
+    needs: run_tests
+    if: github.event_name == 'release'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          # build it on the minimum version
+          python-version: '3.8'
+
+      - name: Upgrade pip
+        run: |
+          pip install --upgrade pip setuptools wheel
+
+      - name: Install swig
+        run: |
+          sudo apt-get update
+          sudo apt-get install libhdf5-serial-dev zlib1g-dev libatlas-base-dev lcov swig3.0 libpcsclite-dev
+          mkdir -p ~/bin/ && ln -s /usr/bin/swig3.0 ~/bin/swig && export PATH=~/bin/:$PATH
+
+      - name: Install and build TAF
+        run: |
+          pip install .[yubikey]
+          python setup.py sdist bdist_wheel clean --all
+
+      - name: Install publishing dependencies
+        run: |
+          pip install packaging
+          pip install twine==3.8.0
+
+      # - name: Upload to TestPyPi
+      #   run: |
+      #     twine check dist/*
+      #     twine upload --repository-url https://test.pypi.org/legacy/ dist/*
+      #   env:
+      #     TWINE_USERNAME: ${{ secrets.TEST_PYPI_USERNAME }}
+      #     TWINE_PASSWORD: ${{ secrets.TEST_PYPI_PASSWORD }}
```

Comment:
Let's remove test pypi code

Reply:
Moved this to docs. I'd like to make sure that we don't have to figure out again how to test the release to PyPi.